packages/core/src/shared/utilities/cacheUtils.ts

```diff
@@ -131,17 +130,16 @@ export function createDiskCache<T, K>(
                 await fs.mkdir(dirname(target))
                 if (isWeb()) {
                     // There is no web-compatible rename() method. So do a regular write.
-                    await fs.writeFile(target, JSON.stringify(data), { mode: 0o600 })
```

Comment:
why are we removing the `mode:` feature?

Reply:
Explained in the second commit of this PR

Reply:
> We do not need these opts since there is only 1 place that uses them, and after re-evaluating they are not needed.

Why aren't they needed? When we create ~/.aws files they need to have restricted permissions. If we aren't doing that, it's a bug.